README.md

```diff
@@ -61,6 +61,7 @@ This is an independent project and we have no affiliation whatsoever with u-blox
 1. Mapview widget with location marker, showing either a static Mercator world map, or an optional dynamic web-based map downloaded via a MapQuest API (*requires an Internet connection and free 
 [MapQuest API Key](https://developer.mapquest.com/user/login/sign-up)*).
 1. Spectrum widget showing spectrum analysis chart from MON-SPAN message (*UBX MON-SPAN messages must be enabled or polled*).
+1. Scatterplot widget showing position with respect to average position.

```

Comment:
prefer something like 'showing variability in position reporting over a time period'

Reply:
Better?